public/partials/report/partials/data-objects.html

```diff
@@ -13,7 +13,8 @@
             <div class="schema-element-report schema-item-report"  
             ng-if="(rootItem.elementRole == 'dimension' || rootItem.elementRole == 'measure')">
 
-                <a id="ELEMENT_{{rootItem.elementID}}" class="list-group-item data-source-element schema-item-report" 
+                <a id="ELEMENT_{{rootItem.elementID}}" class="list-group-item data-source-element schema-item-report"
+                data-toggle="tooltip" data-placement="top" title="{{rootItem.description}}" 
```

Comment:
Same here

Reply:
I did the same here, but I saw that data-placement on this view has a different behavior than " public/partials/layer/view.html " it's a little bit strange but for same behavior I removed useless code here too